tests/v2/validation/provisioning/permutations/permutations.go

```diff
@@ -115,6 +118,42 @@ func RunTestPermutations(s *suite.Suite, testNamePrefix string, client *rancher.
 	}
 }
 
+// RunHostedClusterTestPermutations runs through all relevant perumutations in a given config file.
+func RunHostedClusterTestPermutations(s *suite.Suite, testNamePrefix string, client *rancher.Client, clusterType string) {
```

Comment:
@slickwarren does Hosted clusters even need a permutations function? I don't think we even take a list of k8s verisons for these tests.

Reply:
We don't take a list of K8s versions. I honestly just did this so that we match what Rancher clusters are doing to have consistency in the framework.

Reply:
Then honestly we don't need, our registry tests don't use and our air gap upgrade tests don't either.

Reply:
Fixed